acq4/modules/Imager/Imager.py

```diff
@@ -330,6 +330,10 @@ def __init__(self, manager, name, config):
         # to select from [(dev1, channel1), ...]
         self.detectors = config.get('detectors', [config.get('detector')])
         
+        self.filterwheel = self.manager.getDevice(config['filterwheel'])
```

Comment:
1. Corrected with a 'try' condtion 
2. Changed name to 'filterDevice' and signal to 'sigFilterChanged' to be more general.